src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -6183,7 +6183,7 @@ export interface NS extends Singularity {
    * @param variant - Type of toast, must be one of success, info, warning, error. Defaults to success.
    * @param duration - Duration of toast in ms. Can also be `null` to create a persistent toast. Defaults to 2000
    */
-  toast(msg: any, variant?: string, duration?: number | null): void;
+  toast(msg: any, variant?: "success" | "info" | "warning" | "error", duration?: number | null): void;
```

Comment:
wouldn't this just be type SnackBarVariant? or is it just to make it a bit easier to use from the players perspective?

Reply:
That's exactly my thinking, yeah. Seeing `SnackBarVariant` as the arg type isn't useful if you don't actually have the values of it.